libs/native/src/test/java/org/elasticsearch/nativeaccess/VectorSystemPropertyTests.java

```diff
@@ -27,13 +28,15 @@
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.equalTo;
 
-@WithoutSecurityManager
-public class VectorSystemPropertyTests extends ESTestCase {
+public class VectorSystemPropertyTests extends LuceneTestCase {
```

Comment:
IMO we should keep ESTestCase as the base. Although I know we use LuceneTestCase in some places, having two different base classes makes building common test infrastructure difficult.

Other than the issue of needing to skip windows (and as you say, this wasn't supported on windows anyways), is there something about ESTestCase that inherently makes this test need LuceneTestCase?

Reply:
`tests.system_call_filter` is the recommended approach (along with disabling security manager). For example, we disable syscall filters in the server cli tests since we need to invoke java processes.

Reply:
Ok, cool. I updated the PR to use `tests.system_call_filter`, and keep the anno.